puppet/modules/jenkins_job_builder/files/centos.org/pipelines/lib/duffy.groovy

```diff
@@ -1,21 +1,39 @@
 def provision() {
     dir('foreman-infra') {
-        git url: 'https://github.com/ehelms/foreman-infra.git', branch: 'ci-centos-org'
+        git url: 'https://github.com/theforeman/foreman-infra.git'
```

Comment:
Does this include submodules or not? It can probably be left out here and save some time if it is included.

Reply:
It is needed to provision a duffy box, and I'll need it later for some templates, playbooks, etc.

Reply:
I don't mind the clone, but if it's also cloning submodules (which we only use for puppet) it's doing a lot of work for nothing. We can save time by skipping submodules.

Reply:
@zjhuntin what is a duffy box?

Reply:
duffy is the system powering the CentOS CI infrastructure. We can use it to run our tests. As I understand it's conceptually similar to vagrant or beaker.

Reply:
As I read things, this does not include submodules.